lib/index.coffee

```diff
@@ -10,7 +9,7 @@ setupApp = require './app/index'
 setupPassport = require './passport/index'
 artsyXapp = require 'artsy-xapp'
 
-module.exports = (options) =>
+module.exports = (options) ->
```

Comment:
Subtle... is this a fix or something else?

Reply:
Minor, my vscode coffee linting complains because we don't need `this` for the function to complete.

Reply:
This is also why I added braces in a number of places.